qldbshell/qldb_shell.py

```diff
@@ -151,7 +168,15 @@ def do_EOF(self, line):
 
     @zero_noun_command
     def quit_shell(self, line):
```

Comment:
line parameter is not used

Reply:
I believe these are in-built function signatures of the shell framework.